core/src/main/java/de/danoeh/antennapod/core/util/download/AutoUpdateManager.java

```diff
@@ -1,21 +1,25 @@
 package de.danoeh.antennapod.core.util.download;
 
 import android.util.Log;
+
 import androidx.work.Constraints;
+import androidx.work.Data;
 import androidx.work.ExistingPeriodicWorkPolicy;
 import androidx.work.ExistingWorkPolicy;
 import androidx.work.NetworkType;
 import androidx.work.OneTimeWorkRequest;
 import androidx.work.PeriodicWorkRequest;
 import androidx.work.WorkManager;
-import de.danoeh.antennapod.core.preferences.UserPreferences;
-import de.danoeh.antennapod.core.service.FeedUpdateWorker;
 
 import java.util.Calendar;
 import java.util.concurrent.TimeUnit;
 
+import de.danoeh.antennapod.core.preferences.UserPreferences;
+import de.danoeh.antennapod.core.service.FeedUpdateWorker;
+
 public class AutoUpdateManager {
     private static final String WORK_ID_FEED_UPDATE = FeedUpdateWorker.class.getName();
+    private static final String WORK_ID_FEED_UPDATE_IMMEDIATE = FeedUpdateWorker.class.getName() +"Immediate";
```

Comment:
There is a space missing. Maybe it's better to use a fixed string (instead of using the class name) because then we can't accidentally refactor the name and then have 2 periodic jobs with different names

Reply:
done.